airbyte-webapp/src/packages/cloud/locales/en.json

```diff
@@ -90,6 +90,9 @@
   "settings.generalSettings.changeWorkspace": "Change Workspace",
   "settings.generalSettings.form.name.label": "Workspace name",
   "settings.generalSettings.form.name.placeholder": "Workspace name",
+  "settings.generalSettings.form.advancedMode.label": "Advanced mode",
+  "settings.generalSettings.form.advancedMode.switchLabel": "Enable advanced mode",
+  "settings.generalSettings.form.advancedMode.tooltip": "When Advanced Mode is enabled, certain views will display additional technical information.",
```

Comment:
Is it possible to reuse the strings from the above airbyte-webapp/src/locales/en.json file for Cloud instead of duplicating them here? Maybe not depending on how our packages are set up?